Cargo.toml

```diff
@@ -1,8 +1,16 @@
 [package]
 name = "fastrand-contrib"
+# Make sure to modify CHANGELOG.md before releasing a new version.
 version = "0.1.0"
+authors = ["John Nunley <dev@notgull.net>", "Petr Nevyhoštěný <petr.nevyhosteny@gmail.com>"]
```

Comment:
@pnevyk Used your email from your git commits here, hope that's alright

Reply:
Yeah, it's fine. Thanks for letting me know.